src/ai/backend/storage/utils.py

```diff
@@ -160,8 +160,20 @@ async def log_client_api_entry(
     name: str,
     params: Any,
 ) -> None:
-    log.info(
-        "ClientFacingAPI::{}({})",
-        name.upper(),
-        str(params),
-    )
+    if params is None:
+        log.info(
+            "ClientFacingAPI::{}()",
+            name.upper(),
+        )
+    elif isinstance(params, LoggingInternalMeta):
+        log.info(
+            "ClientFacingAPI::{}({})",
+            name.upper(),
+            params.to_logging_str(),
+        )
+    else:
+        log.info(
+            "ClientFacingAPI::{}({})",
+            name.upper(),
+            str(params),
+        )
```

Comment:
I do have a thought that it makes me wonder if we really need to granularize the actions for logs this much.